### PR TITLE
Move cryptography, keypair and add tests for Helper

### PR DESCRIPTION
**Pull request by ixje**
**What current issue(s) does this address?, or what feature is it adding?**
issue #15 

**How did you solve this problem?**
wip, not finished yet

**How did you make sure your solution works?**
write tests, run tests

**Did you add any tests?**
yes for Helper.py only

**Are there any special changes in the code that we should be aware of?**
yes for `Helper.py`, let's discuss on slack

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -13,3 +13,5 @@ ecdsa==0.13
 base58==0.2.5
 bitcoin==1.1.42
 mpmath==1.0.0
+scrypt==0.8.0
```
Comment (metachris) [inline requirements_dev.txt]: These 2 dependencies need to move into `requirements.txt` which is meant for deps for users of the project. `requirements_dev.txt` is meant for only neocore-dev-specific stuff.

File: neocore/Cryptography/Helper.py [hunk 2]
```diff
@@ -88,7 +71,10 @@ def scripthash_to_address(scripthash):
     Returns:
         str: base58 encoded string representing the wallet address.
     """
-    return bin_to_b58check(binascii.unhexlify(scripthash), 23)
+    sb = bytearray([23]) + scripthash
```
Comment (metachris) [inline neocore/Cryptography/Helper.py]: would love to get the thoughts of @localhuman on this refactoring. looks good to me tho.

**Comment by ixje** [comment]
No not really, the PR adds the following:

first commit
------------
ecdsa==0.13
base58==0.2.5
bitcoin==1.1.42
mpmath==1.0.0

second commit
-------------
scrypt==0.8.0
pycrypto==2.6.1


And finally `cryptography` is a package that's unlikely to be only used in dev. Actually, I'm currently looking into replacing the homebrew `ECCurve.py` with the ECDSA implementation from `cryptography`, so I moved that as well.

**Comment by metachris** [comment]
Understood. Please proceed :+1:

**Comment by metachris** [comment]
I think it looks good. @localhuman any thoughts?

I would proceed to merge this into the feature branch, and then work on that branch with PRs getting the test coverage up, and perhaps doing a few refactorings as suggested by @ixje. 

What do you guys think?

**Comment by localhuman** [comment]
Looks like a good start! Agree with the feature-branch approach. If I was going to replace the ECDSA implementation, I would do it in `neo-python` first so that I can verify it works with all the various signatures throughout the application.

**Pull request merged**